communications.md

```diff
@@ -2,44 +2,79 @@
 
 ## Introduction
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. If done well, we can facilitate great knowledge exchange while still being able to do good work.
+Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. If done well, we can streamline any processes or tools so that they enable us in our creative endeavours, while still facilitating useful knowledge exchange.
 
 ## Tools used for Communication
 
-We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite and Asana.
+We use a few tools to communicate with our customers and teams: Slack, Github, G Suite, Asana and Miro.
```

Comment:
We don't purchase the Miro licence for the whole office. If this doc is facing the whole company, I am not sure whether we should mention Miro or not. But if it's dev only, it's ok.

Reply:
From my understanding, this doc is just for dev at the moment.